src/komponenter/ContextProvider.tsx

```diff
@@ -60,6 +61,7 @@ const ContextProvider = (props: Props) => {
         sistOppdatert,
         settPermitteringInnhold,
         setSideSistOppdatert,
+        dagensDato: new Date(),
```

Comment:
new Date(); tar med klokkaslett, som kan gjøre regninga mellom dager rare ift datoene valgt i DatePicker alltid har klokkaslett ved midnatt (UTC +1) tror jeg. Vil heller at dagens date skal være konstruert. ved å sette dagenDato = new Date() => dagensDato = new Date (dagensdato.getFullYear + '-' dagensDato.getMonth()) osv.

Reply:
Hva med å bruke f eks day.js for dette for å få en "start of day" date?